posthog/urls.py

```diff
@@ -157,6 +157,7 @@ def opt_slash_path(route: str, view: Callable, name: Optional[str] = None) -> UR
     opt_slash_path("capture", capture.get_event),
     opt_slash_path("batch", capture.get_event),
     opt_slash_path("s", capture.get_event),  # session recordings
+    opt_slash_path("p", capture.get_event),  # performance events
```

Comment:
Completely flyby and could warrant its own RFC, but we're growing a pretty big list of urls that point to the same endpoint. We'll need to update [the ingress rules](https://github.com/PostHog/charts-clickhouse/blob/2cd44dd84d7913255b6c53e863a6941f1dda063f/charts/posthog/templates/ingress.yaml) and the [middleware short circuit](https://github.com/PostHog/posthog/pull/13319) to account for this new url.

For operational simplicity (and, anecdotally, for routing performance), could we agree on consolidating all new intake urls into a common `/i/` path (`/p` -> `/i/p`), so than we can easily send every `/i/*` request to the event pool's `get_event` and call it a day?

Longer term, I'd like us to move the existing endpoints too, but that's RFC material for later (how to coordinate SDK and backend releases, although nginx rewrite rules can help address the long tail).